internal/stats/blocks.go

```diff
@@ -39,9 +39,19 @@ func SubmitNewBlockReport(hactarClient *hactar.Client, lotusClient *lotus.Client
 		var blocks []hactar.Block
 		for i, block := range tipset.Blocks {
 			if miner == block.Miner {
+				// determine reward if latest block
+				reward := ""
+				if h == lastHeight {
```

Comment:
In some edge case, this could become different (block mined during this point) but not gonna think about it right now.